spec/helpers/edition_url_helper_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+RSpec.describe EditionUrlHelper, type: :helper do
+  let(:edition) do
+    build(:edition,
+          base_path: "/foo",
+          content_id: "d2547c42-8ed3-49f5-baeb-6112f98c2bf9")
+  end
+
+  describe "#public_url" do
+    it "returns the URL" do
+      url = edition_public_url(edition)
+      expect(url).to eq("https://www.test.gov.uk/foo")
+    end
+
+    it "returns nil without a base_path" do
+      edition = build(:edition, base_path: nil)
+      url = edition_public_url(edition)
+      expect(url).to be_nil
+    end
+  end
+
+  describe "#preview_url" do
+    let(:preview_secret_service) do
+      instance_double("PreviewSecretService", preview_token: "secret")
```

Comment:
Looks like this needs to be updated

Reply:
Ah, thanks for spotting - I've added an extra commit to help prevent this happening again. Could you take a look?